src/server/apis/dash-data.ts

```diff
@@ -4,15 +4,16 @@ import gql from 'graphql-tag';
 import * as api from '../../types/api';
 import {mentionedFieldsFragment, prFieldsFragment, PullRequestReviewState, reviewFieldsFragment, ViewerPullRequestsQuery} from '../../types/gql-types';
 import {github} from '../../utils/github';
-import {userModel} from '../models/userModel';
+import {ID_COOKIE_NAME, userModel} from '../models/userModel';
 
 export class DashData {
   getHandler() {
     return this.handler.bind(this);
   }
 
   private async handler(req: express.Request, res: express.Response) {
-    const loginDetails = await userModel.getLoginFromRequest(req);
+    const loginDetails =
```

Comment:
It might make sense to add a new API function to `userModel` `getLoginFromRequest()` which just factors out reading from the cookie and calling `getLoginFromToken()`. WDYT?